bedrock/mozorg/templates/mozorg/MPL/2.0/combining-mpl-and-gpl.html

```diff
@@ -0,0 +1,120 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/base-resp.html" %}
+
+{% block page_title %}{{ _('MPL-in-GPL Developer Guidelines') }}{% endblock %}
+
+{% block page_desc %}{{ _('Combining MPL-Licensed files with an &#40;L&#41;GPL-Licensed Project: Guidelines for Developers') }}{% endblock %}
+
+{% block page_css %}
+    {% stylesheet 'mpl' %}
+{% endblock %}
+
+{% block content %}
+
+<div id="main-content">
+    <h1>Combining MPL-Licensed files with an (L)GPL-Licensed Project: Guidelines for Developers</h1>
+
+    <section>
+        <h2 id="introduction">Introduction</h2>
+        <p>When someone combines a file or files licensed under the <a href="{{ url('mozorg.MPL.2.0.index') }}">Mozilla Public License, version 2.0 ("MPL")</a> with a project licensed under the GNU General Public License or Lesser General Public Licenses ("(L)GPL"), the MPL's <a href="{{ url('mozorg.MPL.2.0.index') }}#distribution-of-a-larger-work">Section 3.3</a> allows distribution of the combined work (the "Larger Work") subject to the terms of both licenses, as long as certain conditions are met.</p>
+        <p>This document is intended to help developers take advantage of this provision, while still complying with the MPL and (L)GPL and respecting the intent of upstream developers.</p>
+        
+        <h2 id="initial-distribution-of-mpl-licensed-files-as-part-of-an-lgpl-licensed-project">Initial Distribution of MPL-licensed Files As Part Of an (L)GPL-licensed Project</h2>
+        <p>When a developer combines MPL-licensed files into an (L)GPL-licensed project for initial distribution under both MPL and (L)GPL, the MPL-licensed files may be redistributed under one of three different circumstances: they may be unmodified; they may be unmodified, but policy requires or the developer prefers to add an (L)GPL header; or they may be modified.</p>
+        <h3 id="unmodified-mpl-licensed-files-mpl-only">Unmodified MPL-licensed Files - MPL-only</h3>
+        <p>In the simplest case, the developer combines unmodified MPL-licensed files into a project with (L)GPL-licensed files to create a Larger Work, and seeks to distribute the resulting combination under the terms of the (L)GPL.</p> 
+        <p>In this case, Section 3.3 of the MPL permits the individual files to be distributed as part of the Larger Work, with no further changes required. The developer may simply leave the file untouched, with all notices intact. The top of the incorporated file would continue to look like <a href="{{ url('mozorg.MPL.2.0.index') }}#exhibit-a---source-code-form-license-notice"> Exhibit A</a> of the MPL:</p> 
+        <pre>
+        This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Another spot where source indenting is affecting the layout of the page in `<pre>` tags.